libs/api/metadata-converter/src/lib/gn4/gn4.field.mapper.ts

```diff
@@ -291,7 +291,7 @@ export class Gn4FieldMapper {
             <SourceWithUnknownProps>selectField(source, 'related'),
             'fcats'
           )
-        ),
+        ) ?? {},
```

Comment:
Just wondering, shouldn't this be the atomic operation's responsbility?

Reply:
I was wondering about that too, but then that would mean we can't check on the value being defined anymore. `getFirstValue` does what it says, without any changes to the value.